operations/rates.md

```diff
@@ -116,6 +116,7 @@ Note this operation uses [Pagination](../guidelines/pagination.md) and supports
 | `IsActive` | boolean | required | Whether the rate is still active. |
 | `IsEnabled` | boolean | required | Whether the rate is currently available to customers. |
 | `IsPublic` | boolean | required | Whether the rate is publicly available. |
+| `Type` | string | required | Type of the rate (Public, Private, AvailabilityBlock, etc.) |
 | `Name` | string | required | Name of the rate. |
```

Comment:
maybe we can add the description here directly in this pull request, since we are just adding it. Thanks Britta :)